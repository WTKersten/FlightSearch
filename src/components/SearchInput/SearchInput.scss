.search-input {
  font-size: 16px;
  padding: 5px;
  width: 300px;
}