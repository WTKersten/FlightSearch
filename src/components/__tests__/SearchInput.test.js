import React from 'react';
import { shallow } from 'enzyme';
import SearchInput from '../SearchInput/SearchInput';

describe('SearchInput', () => {
  it('renders correctly', () => {
    const component = shallow(<SearchInput />);
    expect(component).toMatchSnapshot();
  });
});