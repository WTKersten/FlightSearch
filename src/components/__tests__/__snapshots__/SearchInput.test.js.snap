// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInput renders correctly 1`] = `
<div
  className="component-search-input"
>
  <div>
    <input
      className="search-input"
      onChange={[Function]}
    />
  </div>
</div>
`;
